### ISSUE TITLE
Show a warning if any title attribute is detected (except on iframe)

### ISSUE THREAD

**Issue by jackdomleo7** [jackdomleo7/Checka11y.css#96]
### Describe the new a11y feature or project enhancement

Show a warning if any `title` attribute is detected. `title` has [many accessibility issues](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns) and should be replaced with a [more accessible technique](https://inclusive-components.design/tooltips-toggletips). This rule should **not** apply to the `<iframe>`, since `title` is required on `<iframe>` to describe its content.

### Describe the solution you'd like

```scss
[title] {
  // Show warning
}

iframe[title] {
  // Remove warning
}
```

### Link(s)

- https://www.a11yproject.com/checklist/#remove-lesscodegreatertitleless/codegreater-attribute-tooltips
- https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns
- https://inclusive-components.design/tooltips-toggletips
